packages/teams-js/src/internal/telemetry.ts

```diff
@@ -198,7 +198,8 @@ export const enum ApiName {
   Menus_SetNavBarMenu = 'menus.setNavBarMenu',
   Menus_SetUpViews = 'menus.setUpViews',
   Menus_ShowActionMenu = 'menus.showActionMenu',
-  MessageChannels_GetTelemetryPort = 'messageChannels.getTelemetryPort',
+  MessageChannels_Telemetry_GetTelemetryPort = 'messageChannels.telemetry.getTelemetryPort',
```

Comment:
Also, can you explain how will the telemetry will be handled from previous `messageChannels.getTelemetryPort` to `messageChannels.telemetry.getTelemetryPort`? Can't we use the existing one?